running-your-first-ot-server/setup-webserver-linux.md

```diff
@@ -0,0 +1,201 @@
+# Getting started
+Rent a Ubuntu Server 18.04 or higher from a trusted hosting company such or install for the sake of learning experience Ubuntu Server 18.04 or higher in VirtualBox.
+
+Once you're set we will start by accessing our server. If you're on windows you should get putty(https://www.putty.org/), if you're on macOS or Linux you can use the terminal which is standard available. With putty just follow the screen ( insert your domain/ip-address and click ok ) for macOS and linux users type:
+
+```bash
+ssh username@domain/ipaddress
+```
+`username example => root
+domain/ipaddress example => yourdomain.com or 127.0.0.1`
+
+Now you will be prompted for your password (if in putty you will be prompted for the password as soon you click on Open) Insert your password and presh enter (you will not see what you type, so make sure you type your password in the right order). Now we are in our server :)
+
+## Update and Upgrade the system
+The first thing we will do is updating our Ubuntu Server with the following command
+```bash
+sudo apt update && sudo apt upgrade -y
+```
+We are using 'sudo' since I do not know if you're running as root user or as regular user. When you're logged in as regular user. No problem! Just enter your password when the terminal asks for your password. With the command above you will update the connected repository's the server is connected to and when the server is done updating the repository's it will directly upgrade all the packages which were updated.
+
+# Installing the webserver
+
+Now we will get to the more difficult part of the entire server setup, installing nginx, php-fpm, mariadb-server with phpmyadmin.
+```bash
+sudo apt install nginx php-fpm mariadb-server -y
+```
+The installation process is now triggered.When this is finished we will install phpmyadmin to handle our database server (which is mariadb)
+```bash
+sudo apt install phpmyadmin -y
+```
+Now we are set to get some configurations going!
+
+## Configuring the webserver
+First we want to know which version of php-fpm is installed we will check this by writing the following command in the terminal
+```bash
+sudo php -v
+```
+The output will be looking something similar as this (I am using ubuntu 19.10 here which uses php version 7.3 as default. If you're running on a server you most likely run on 18.04 lts or with the newest version 20.04 lts):
+
+![alt text](https://worldofcoding.net/github-img/otland-gitbook/phpversion.png "phpversion")
```

Comment:
what is this website? It should ideally be just text, let's avoid images of text.